.gitignore

```diff
@@ -1,13 +1,18 @@
 .improbable/
 .idea/
 spatialos.*.build.json
-packages/
 logs/
 
-SpatialOS/schema/bin/
+SpatialOS/schema_out
```

Comment:
I'm not sure if I'm jut visualising this the wrong way, but I think you might still have to rename the existing `schema` folder to `schema_out`

Reply:
if I understand correctly, all generated schema-related files are now either in `shcema_out` or each worker's folder, so don't need to clean `schema/` after build. 🤔

Reply:
Generated code will be in the individual worker dirs under src/improbable/generated. The schema descriptor will be under <top level>/schema_out. No bin folder in schema_out. Sorry for the last minute change.

Reply:
thanks for clarifying :D